README_KOR.md

```diff
@@ -0,0 +1,197 @@
+- 다양한 게임 모드 (featuring GetTheChicken, Domination, CTF, ...)
+- online and local multiplayer온라인과 로컬 멀티플레이
+- Comes with the leveleditor레벨 에디터와 함께 제공 - 자신만의 맵 제작...
+- ... 그리고 많은 사람들의 그들만의 맵 제작. 이로 인해 현재 맵은 1000개를 넘어간다
+- 막대기로 원숭이를 찌르는 것들보다 훨씬 재밌다(텀블링 몽키?)
+- 모든 게임에 관련된 소스코드는 무료로 이용가능하
+- SDL을 사용하여 windows, linux, mac 등에 모두 호환된다
+- CPU 플레이어와 함께 할 수 있다
+- 당신을 즐겁게 할 것이다!!!
+
+### Supported platforms
+
+- Linux
+- Windows
+- Mac OS X
+- ARM devices (예를 들어 Raspberry Pi-라즈베리 파이)
+- XBox (?)
+- Android (실험중)
+- asm.js (실험중)
+
+
+## Building instructions
+
+### Requirements
+
+- C++11를 지원하는 컴파일러 (예 gcc-4.8)
+- CMake (>= 2.6)
+- SDL (1.2 or 2.0) 그리고,
+    - SDL_image
+    - SDL_mixer
+- zlib
+- yaml-cpp (포함됨)
+- ENet (선택, 포함됨)
+
+저런 조건을 위해 패키지 매니저를 사용 할 수 있다
+각 시스템 별 코드:
+
+- Debian-based: `apt-get install cmake libsdl1.2-dev libsdl-image1.2-dev libsdl-mixer1.2-dev zlib1g-dev`
+- Fedora/RPM: `yum install cmake SDL-devel SDL_image-devel SDL_mixer-devel zlib-devel`
+- Arch: `pacman -S cmake sdl sdl_image sdl_mixer zlib`
+- MSYS2: `pacman -S mingw-w64-x86_64-SDL mingw-w64-x86_64-SDL_image mingw-w64-x86_64-SDL_mixer mingw-w64-x86_64-zlib`
+
+다른 시스템들을 사용한다면 아래의 주소에서 개발파일을 수동 다운로드 :
+
+- http://www.cmake.org/cmake/resources/software.html#latest
+- http://www.libsdl.org/download-1.2.php
+- http://www.libsdl.org/projects/SDL_image/release-1.2.html
+- http://www.libsdl.org/projects/SDL_mixer/release-1.2.html
+- http://zlib.net
+
+### Get the code
+
+이 저장소에는 당신의 운영체제와 맞지 않거나 당신이 시스템에 설치하는 것을 원하지 않는 경우들에 사용할 수 있는 일부 하위 모듈들을 포함하고 있다. 포함되어 있는 라이브러리들을 이용하면 재귀적인 복사를 할 수 있을 것이다:
+
+`git clone --recursive https://github.com/mmatyas/supermariowar.git`
+
+또는 하위 모듈을 수동으로 초기화도 가능하다:
+
+```sh
+git clone https://github.com/mmatyas/supermariowar.git
+cd supermariowar
+git submodule update --init
+```
+
+오히려 시스템 라이브러리를 사용하려는 경우, [Build configuration](#build-configuration) 의 이 기능을 비활성화 시키는 부분을 참고할 수 있다.
+
+### Linux
+
+빌드할 디렉토리를 작성하고 CMake를 실행하여 프로젝트를 구성한다. 그리고 빌드하려고 할 때마다 `make`를 호출해라. 바이너리는 기본적으로 `./build/Binaries/Release` 에 생성된다. 간단히 말하자면:
+
+```sh
+unzip data.zip
+mkdir build && cd build
+cmake ..
+make -j4 # -jN = build on N threads
+./Binaries/Release/smw ../data
+```
+
+`make`의 주요 빌드 대상:
+
+- smw
+- smw-leveleditor
+- smw-worldeditor
+- smw-server
+
+당신이 IDE에서 개발하기를 더 선호한다면 (CodeBlocks, Eclipse, ...), CMake를 활용해서 프로젝트 파일을 생성할 수 있을 것이다. 여기서 더 많은 정보를 찾을 수 있을 것이다 [Build configuration](#build-configuration).
+
+설치 가능한 패키지를 만들려면 `make package`를 실행하라. 그러면 TGZ, DEB and RPM 파일들이 만들어 질 것이다.
+
+### Mac OSX
+
+당신은 아마 Xcode와 command line 툴들이 필요 할 것이다; 사이트에서 SDL과 CMake를 수동으로 설치하거나 Homebrew를 통해 받을 수 있다: `brew install cmake sdl2 sdl2_image sdl2_mixer`. 그리고나서 Linux의 설명을 따라 Super Mario War 를 빌드하면 된다.
+
+### Windows
+
+만약 당신이 MinGW Shell/MSYS 와 Cygwin를 사용한다면, Linux 가이드를 따르면 된다. 당신은 CodeBlocks, Eclipse, Visual Studio 같은 많은 IDE들을 CMake를 이용하여 프로젝트 파일을 생성할 수도 있다.
+
+더 자세한 것은 wiki에서 : [Building on Windows](https://github.com/mmatyas/supermariowar/wiki/Building-on-Windows).
```

Comment:
Is a space necessary before `:`?

Reply:
It is not necessary. remove it for unity